src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexRunner.cs

```diff
@@ -22,50 +22,78 @@ namespace System.Text.RegularExpressions
     [EditorBrowsable(EditorBrowsableState.Never)]
     public abstract class RegexRunner
     {
-        protected internal int runtextbeg;         // Beginning of text to search. We now always use a sliced span of the input
+        /// <summary> Beginning of text to search </summary>
+        protected internal int runtextbeg;         // We now always use a sliced span of the input
```

Comment:
As long as you're converted comments into XML comments, can you move all of these comments? These would go as remarks.